documentation/modules/oauth/con-oauth-authentication-flow.adoc

```diff
@@ -65,28 +81,21 @@ When used with the OAuth 2.0 over PLAIN callbacks, Kafka clients authenticate wi
 
 * A long-lived access token, obtained manually at configuration time
 
-The client must be enabled to use PLAIN authentication, and provide a `username` and `password`.
-If the password is prefixed with `$accessToken:` followed by the value of the access token, the Kafka broker will interpret the password as the access token.
-Otherwise, the Kafka broker will interpret the `username` as the client ID and the `password` as the client secret.
+For both methods, the client must provide the PLAIN `username` and `password` properties to pass credentials to the Kafka broker.
+The client uses these properties to pass a client ID and secret or username and access token.
+A client ID and secret are used to obtain an access token.
 
-If the `password` is set as the access token, the `username` must be set to the same principal name that the Kafka broker obtains from the access token.
-The process depends on how you configure username extraction using `userNameClaim`, `fallbackUserNameClaim`, `fallbackUsernamePrefix`, or `userInfoEndpointUri`.
-It also depends on your authorization server; in particular, how it maps client IDs to account names.
+For the Kafka broker to interpret a password as an access token, the following configuration is required:
 
-To use PLAIN, you must enable it in the `oauth` listener configuration for the Kafka broker.
+* The broker listener configuration must have a token endpoint (`tokenEndpointUri`) configured
+* The password requires an `$accessToken:` prefix followed by the value of the access token
 
-In the following example, PLAIN is enabled in addition to OAUTHBEARER, which is enabled by default.
-If you want to use PLAIN only, you can disable OAUTHBEARER by setting `enableOauthBearer` to `false`.
+If a token endpoint is not configured on the broker listener, the Kafka broker interprets the password as a raw access token.
+In this case, the password should not include a prefix.
 
-[source,yaml,subs="+quotes,attributes+"]
-----
-  # ...
-  authentication:
-    type: oauth
-    # ...
-    enablePlain: true
-    tokenEndpointUri: https://_OAUTH-SERVER-ADDRESS_/auth/realms/external/protocol/openid-connect/token
-----
+If the `password` is set as the access token, the `username` must be set to the same principal name that the Kafka broker obtains from the access token.
+The process depends on how you configure username extraction using `userNameClaim`, `fallbackUserNameClaim`, `fallbackUsernamePrefix`, or `userInfoEndpointUri`.
+It also depends on your authorization server; in particular, how it maps client IDs to account names.
```

Comment:
'It' should be specified. Sentence is vague

Reply:
Changed to:

> You can specify username extraction options in your listener using the `userNameClaim`, `fallbackUserNameClaim`, `fallbackUsernamePrefix`, and `userInfoEndpointUri` properties.
> The username extraction process also depends on your authorization server; in particular, how it maps client IDs to account names.